var/spack/repos/builtin/packages/openspeedshop/package.py

```diff
@@ -83,8 +77,8 @@ class Openspeedshop(Package):
                          to point to target build.")
     variant('cuda', default=False,
             description="build with cuda packages included.")
-    variant('ptgf', default=False,
-            description="build with the PTGF based gui package enabled.")
+    variant('noqt3gui', default=False,
+            description="build without the build Qt3 gui package enabled.")
```

Comment:
We have a qt4/qt5 gui as well.  That is a new package I'm working on adding.  If I don't turn off the qt3 gui build, I get a conflict when building the qt4/qt5 gui package (cbtf-argonavis-gui, which is not checked in yet).   The default is to have the qt3 gui built.  That is why I made it a no qt3 gui option.